packages/components/src/Image/docs/size.tsx

```diff
@@ -0,0 +1,11 @@
+import { Image, Inline } from "@hopper-ui/components";
+
+export default function Example() {
+    return (
+        <Inline>
+            <Image src="/frog.jpg" alt="Frog" width="core_640" height="auto" />
```

Comment:
Maybe we should show examples with the UNSAFE_width instead? most of the time, images aren't the size of tokens anyway. The main use case will be UNSAFE_width="1174px" most likely

Reply:
Feels weird to expose that we want to use something UNSAFE in the documentation. But I get it for this case.